lasagna-master/ena/lasagna_master.go

```diff
@@ -0,0 +1,38 @@
+package lasagna
+
+func PreparationTime(layers []string, timePerLayer int) int {
+	const averageTimePerLayer = 2
+	if timePerLayer == 0 {
+		timePerLayer = averageTimePerLayer
+	}
+	return len(layers) * timePerLayer
+}
+
+func Quantities(layers []string) (int, float64) {
+	const noodlesPerLayer = 50 // grams
+	const saucePerLayer = 0.2  // liters
+	noodlesLayers := 0
+	sauceLayers := 0
+	for _, entry := range layers {
+		if entry == "sauce" {
+			sauceLayers++
+		} else if entry == "noodles" {
+			noodlesLayers++
+		}
+	}
+	return noodlesLayers * noodlesPerLayer, float64(sauceLayers) * saucePerLayer
+}
+
+func AddSecretIngredient(friendsList, myList []string) []string {
+	ret := myList
+	magicIngredient := friendsList[len(friendsList)-1]
+	return append(ret, magicIngredient)
```

Comment:
This code is too verbose in my opinion. I think it's cleaner to have something like:
```
return append(myList, friendsList[len(friendsList)-1])
```

Reply:
Yes, I updated that just after I sent it, but you were too quick.